src/components/AppLayout/NavHeader.tsx

```diff
@@ -117,9 +119,9 @@ const NavHeader = ({ className, sidedrawer, setSidedrawer, displayName, isVerifi
 					href='https://townhallgov.com/'
 					target='_blank'
 					rel='noreferrer'
-					className='custom-link'
+					className='custom-link after:hidden'
```

Comment:
why we are using this after:hidden?

Reply:
The after element is absolute so the hover does not work on the menu item.